src/routers/memos/index.ts

```diff
@@ -0,0 +1,59 @@
+import createRouter from '@/utilities/functions/createRouter';
+import { StatusCode } from '@/models/statusCodes';
+
+const router = createRouter();
+
+/**
+ * @api {get} /memos Get All Memos
+ * @apiName GetMemos
+ * @apiGroup Memos
+ * @apiVersion  0.0.1
+ * @apiDescription Returns all memos.
+ *
+ * @apiError (403) Forbidden Current user type does not have sufficient privileges.
+ *
+ * @apiSuccess (200) {String} Success Successfully retrieved all memos
+ * @apiSuccessExample /memos Success-Response:
+  [
+    {
+      "memoUuid": "1e167b81-d816-497b-8c0c-36f4d6b2fd33",
+      "authorUuid": "425825a2-1e6f-4cfb-b612-e9933bf6c28e"
```

Comment:
Do we want to also return the name here too like we've done for the other calls?